src/models/v2/channel.ts

```diff
@@ -53,8 +54,9 @@ export class Channel extends BaseModel<v2.ChannelObject, { id: string, address:
   operations(): OperationsInterface {
     const operations: OperationInterface[] = [];
     ['publish', 'subscribe'].forEach(operationAction => {
+      let id =  this._json[operationAction as 'publish' | 'subscribe'] && (this._json[operationAction as 'publish' | 'subscribe'] as OperationObject).operationId || this.meta().id + "_" + operationAction;
```

Comment:
You don't reassign that variable, so we can make it const:

```suggestion
      const id =  this._json[operationAction as 'publish' | 'subscribe'] && (this._json[operationAction as 'publish' | 'subscribe'] as OperationObject).operationId || this.meta().id + "_" + operationAction;
```

Reply:
Yeah, it was a `let` because previously I didn't add a logical operator in there but an if later. Good catch!